@stellar/design-system-website/docs/intro.mdx

```diff
@@ -13,7 +19,19 @@ sidebar_position: 1
 </PreviewBlock>
 ```
 
-Let's discover **Docusaurus in less than 5 minutes**.
+<ComponentProps componentName="Button" />
+
+## SDS Link component
+
+<ComponentDescription componentName="Link" />
+
+```tsx live
+<PreviewBlock>
+  <Link>Test link</Link>
+</PreviewBlock>
+```
+
+<ComponentProps componentName="Link" />
```

Comment:
We can make this a whole component, but we can figure it out once we create actual component pages.